nixos/modules/services/backup/tarsnap.nix

```diff
@@ -258,12 +272,13 @@ in
       serviceConfig = {
         IOSchedulingClass = "idle";
         NoNewPrivileges = "true";
-        CapabilityBoundingSet = "CAP_DAC_READ_SEARCH";
+        CapabilityBoundingSet = "CAP_DAC_READ_SEARCH CAP_NET_RAW";
```

Comment:
I think you can do away with this if you set `PermissionsStartOnly = true`; then, restrictions only apply to the process launched by `ExecStart`.